src/main/java/seedu/address/model/activity/Activity.java

```diff
@@ -0,0 +1,103 @@
+package seedu.address.model.activity;
+
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+
+import java.util.ArrayList;
+
+import seedu.address.model.person.Person;
+
+/**
+ * Represents an Activity class containing participants and expenses.
+ */
+public class Activity {
+
+    private final String title;
+    private final ArrayList<Person> participants = new ArrayList<>();
+    private final ArrayList<Expense> expenses = new ArrayList<>();
+
+    /**
+     * Constructor for Activity.
+     * @param title A String representation of the activity name.
+     * @param people The people participating in the activity.
+     */
+    public Activity(String title, Person ... people) {
+        requireAllNonNull(title);
+        this.title = title;
+        for (int i = 0; i < people.length; i++) {
+            participants.add(people[i]);
+        }
+    }
+
+    /**
+     * Gets the list of participants in the activity.
+     * @return An ArrayList containing the participants.
+     */
+    public ArrayList<Person> getParticipants() {
+        return participants;
+    }
+
+    /**
+     * Gets the list of expenses in the activity.
+     * @return An ArrayList of expenses.
+     */
+    public ArrayList<Expense> getExpenses() {
+        return expenses;
+    }
+
+    /**
+     * Gets the name of the activity.
+     * @return A String representation of the name of the activity.
+     */
+    public String getTitle() {
+        return title;
+    }
+
+    /**
+     * Invite people to the activity.
+     * @param people The people that will be added into the activity.
+     */
+    public void invite(Person ... people) {
+        for (int i = 0; i < people.length; i++) {
+            // haven't implement check for person with same name
```

Comment:
Add a `// TODO:` prefix to incomplete parts that have not been implemented so it is easier for us to search them up later, e.g. `// TODO: implement check for unique names`.